frontend/src/scenes/insights/Insight.tsx

```diff
@@ -163,8 +167,25 @@ export function Insight({ insightId }: { insightId: InsightShortId | 'new' }): J
                         {insightMode !== ItemMode.Edit && (
                             <>
                                 <More
+                                    tooltip={
```

Comment:
You can just wrap this in `Tooltip`. If the tooltip title is falsy, the child component is simply rendered directly.